src/ucs/profile/profile_defs.h

```diff
@@ -116,26 +116,80 @@ typedef struct ucs_profile_record {
     uint32_t                 location;      /**< Location identifier */
 } UCS_S_PACKED ucs_profile_record_t;
 
+typedef struct ucs_profile_global_location {
```

Comment:
1. 2 space lines in H file
2. do we really need to expose these structs n H file?  maybe add function which allocates and initializes a profile context, e.g ucs_profile_context_create(mode,flags,etc/) (and destroy)